§Docs (#2)
animo/aries-framework-javascript: docs/libindy/android.md

# Setup Libindy for Android

> > NOTE: If these steps do not work, or you want to do a manual install, refer to [here](https://github.com/hyperledger/indy-sdk#android)

Reviewer: ```suggestion
> NOTE: If these steps do not work, or you want to do a manual install, refer to the [Indy SDK docs](https://github.com/hyperledger/indy-sdk#android)
```

Reviewer: dit ook in de andere files

## Android

1. Go to the [release page](https://github.com/animo/libindy-binaries) and download the Android binaries

>    - These assets are acquired from [here](https://repo.sovrin.org/android/libindy/stable/1.16.0/)

Reviewer: ```suggestion
   - These assets are acquired from the [Sovrin repo](https://repo.sovrin.org/android/libindy/stable/1.16.0/)
```

Reviewer: I.p.v. `here` the gebruiken voor links probeer ik altijd te omschrijven waar de link heen gaat

> 2. Move the `jniLibs` to the `android/app/src/main/`

Reviewer: ```suggestion
2. Move the `jniLibs` directory to the `android/app/src/main/ directory`
```

3. set the minimal SDK version in `android/build.gradle`

> ```json

Reviewer: ```suggestion
```gradle
```

Reviewer: je hebt op meerdere plekken json gezet waar gradle moet staan
